### PR TITLE
Write the code that can take the paper wallet PDF and parse it back into the JSON

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] Moved loading screen to new routes
- [x] Moved RCP to new routes
- [x] Fixed header is not showing in the new UI
- [x] Updated UI for PDF 
- [x] Implement the import feature from PDF
- [x] Update UI for restore popup

##### FreedCamp
- [x] [Write the code that can take the paper wallet PDF and parse it back into the JSON](https://freedcamp.com/view/2956183/tasks/40935261)

### CODE REVIEW

**Review by sanjanadhikari1 — APPROVED** [review]
Looks good for me!

File: src/features/profile/mySecurity/restorePrivateKeyAndPastelID/RestoreModal.tsx [hunk 1]
```diff
@@ -35,8 +40,14 @@ export default function RestoreModal({
       onClose={() => handleCloseModal(false)}
       closeButton
       render={() => (
-        <div className='paper p-10 w-[690px]'>
-          <div className='py-5'>
+        <div
+          className={cn(
+            'paper p-10',
+            selectedRestoreType && 'w-[690px]',
+            !selectedRestoreType && 'w-[791px]',
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/features/profile/mySecurity/restorePrivateKeyAndPastelID/RestoreModal.tsx]: Updated

File: src/common/routes/Routes.tsx [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+  disconnected: boolean
+  latestBlock: number
+  pslPrice: number | undefined
+  solps: number
+  testnet: boolean
+  verificationProgress: number
+  version: number
+}
 
 type RouteType = {
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/common/routes/Routes.tsx]: Updated

**Review by goal-shevchenko — APPROVED** [review]

**Comment by goal-shevchenko** [comment]
Looks good to me

**Pull request merged**